x-pack/plugins/maps/ui_tests/fixtures/index.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  test as base,
+  PageObjects,
+  createLazyPageObject,
+  ScoutTestFixtures,
+  ScoutWorkerFixtures,
+} from '@kbn/scout';
+import { GisPage } from './page_objects';
```

Comment:
I think it is a good candidate to go directly in `kbn-scout` as other plugins will be using maps (dashboard, ml, etc)
Another thought: `maps` is its alias in FTR, maybe we can call it `MapsPage`?

Reply:
I agree to both suggestions.

Reply:
I was told the presentation team prefers this naming, but I can double check

Reply:
I pinged the team about the naming, waiting to hear back